pkg/Library/Graph/Atoms/GraphStatus.js

```diff
@@ -4,23 +4,16 @@ export const atom = (log, resolve) => ({
  * Copyright 2023 NeonFlan LLC
  * SPDX-License-Identifier: BSD-3-Clause
  */
-initialize(_, state, {invalidate}) {
-  timeout(() => {
-    state.waited = true;
-    invalidate()
-  }, 1000);
-},
 render({graph}, {waited}) {
   let warning = null;
   if (graph?.meta?.readonly) {
     warning = 'This graph is readonly. Clone it, if you want to modify it (or your changes won\'t be saved).';
   } else if (!graph) {
     warning = 'Please, select a graph';
   }
-  warning = warning && waited;
   return {
     warning,
-    hideWarning: String(!warning)
+    showWarning: String(Boolean(warning))
```

Comment:
IIUC you added this to avoid the warning appearing, and then quickly disappearing... but i think, changing `hide$=` to `show$=` also fixes it.. or am i wrong?

Reply:
I donno, does it matter? Feel free to change it however.